src/main/java/picard/arrays/illumina/InfiniumGTCFile.java

```diff
@@ -163,12 +169,17 @@ public void close() throws IOException {
         stream.close();
     }
 
-    InfiniumGTCFile(final DataInputStream gtcStream) throws IOException {
-        super(gtcStream, true);
-        this.normalizationManifest = null;
-        parse();
-    }
+    private void loadNormalizationIds(final File bpmFile) {
+        IlluminaBPMFile illuminaBPMFile;
+        try (final DataInputStream inputStream = new DataInputStream(new FileInputStream(bpmFile))) {
+            illuminaBPMFile = new IlluminaBPMFile(inputStream);
```

Comment:
```suggestion
            illuminaBPMFile = new IlluminaBPMFile(bpmFile);
```
And make IlluminaBPMFile autocloseable

Reply:
yup